### ISSUE TITLE
Allow custom change types

### ISSUE THREAD

**Issue by EiyuuZack** [brightcove/kacl#6]
Some projects adapt change types to fit their needs (e.g using present tense _Fixes_ instead of _Fixed_ or adding an _Improved_ change type).

My suggestion for a possible quick solution would be having a `.kacl` file in `$HOME` containing a list of change types such as:
> Adds
> Changes
> Breaking Changes
> Improves
> Fixes
> Deprecates
> Removes

This file -- if present -- would be read by `kacl` and overwrite the default change types during lint checks.

I believe this would be a nice feature to have as it provides a good degree of flexibility.

Keep up the awesome work!

**Comment by tjaneczko** [comment]
Good idea!  This project uses https://github.com/oscarotero/keep-a-changelog/ for handling the parsing and manipulating of changelogs, so we'd need to add support for this in there.  Currently it's hardcoded here: https://github.com/oscarotero/keep-a-changelog/blob/master/src/Release.js#L17 , but that might be possible to change pretty easily.

**Comment by stropho** [comment]
and now `keep-a-changelog` project enables users to add custom types :smiley_cat:
